dbt/adapters/athena/lakeformation.py

```diff
@@ -20,10 +22,42 @@
 logger = AdapterLogger("AthenaLakeFormation")
 
 
+def _column_tags_to_remove(
```

Comment:
WDYT about moving these 2 methods to `LfTagsManager` as staticmethods ?

Reply:
Sure, done